CMakeLists.txt

```diff
@@ -794,6 +794,10 @@ if(USE_COREML_DELEGATE)
   string(APPEND CMAKE_CXX_FLAGS " -DUSE_COREML_DELEGATE")
 endif()
 
+if(BUILD_CAFFE2)
```

Comment:
This reverted in #91949 . We can avoid changing in this PR, or merge both PRs into one?

Reply:
Sure, merged these two PRs into 1 https://github.com/pytorch/pytorch/pull/94163. Thanks for advice.